migrations/V107__Add_Role_and_Component_to_System_Intake_Contacts.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE system_intake_contacts ADD COLUMN component text;
```

Comment:
Should these be non-nullable? I'm pretty sure the user will always be selecting from the list of components/roles we provide.